cartography/data/jobs/analysis/aws_ec2_asset_exposure.json

```diff
@@ -1,34 +1,66 @@
 {
 	"statements": [
 		{
-			"query": "MATCH ((n) <-[:RESOURCE]-(:AWSAccount)<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID})) where EXISTS(n.exposed_internet) AND labels(n) IN ['AutoScalingGroup', 'EC2Instance', 'LoadBalancer', 'LoadBalancerV2'] WITH n LIMIT $LIMIT_SIZE REMOVE n.exposed_internet, n.exposed_internet_type return COUNT(*) as TotalCompleted",
```

Comment:
we should not remove cloudanix workspace check here. for this version of code, it's needed.